### PR TITLE
kubic: Add kubeadm role installation

### PR DESCRIPTION
**Pull request by kravciak**
Local runs:
 - http://dhcp126.suse.cz/tests/10664 - kubic kubeadm
 - http://dhcp126.suse.cz/tests/10665 - kubic microos
 - http://dhcp126.suse.cz/tests/10663 - caasp admin

Needles: https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/467

### CODE REVIEW

File: tests/installation/kubeadm_settings.pm [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# Maintainer: Martin Kravec <mkravec@suse.com>
+
+use strict;
+use warnings;
+use base "y2logsstep";
+use testapi;
+
+sub run {
+    assert_screen 'kubeadm-settings';
+    check_screen 'kubeadm-ntp-empty';    # bsc#1114818
```
Comment (sysrich) [inline tests/installation/kubeadm_settings.pm]:
```suggestion
    if (check_screen 'kubeadm-ntp-empty', 0) {
        record_soft_failure 'bsc#1114818';
```

File: tests/installation/kubeadm_settings.pm [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    send_key 'alt-t';
```
Comment (sysrich) [inline tests/installation/kubeadm_settings.pm]:
```suggestion
        send_key 'alt-t';
```

File: tests/installation/kubeadm_settings.pm [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    type_string '0.opensuse.pool.ntp.org';
```
Comment (sysrich) [inline tests/installation/kubeadm_settings.pm]:
```suggestion
        type_string '0.opensuse.pool.ntp.org';
    }
```

File: tests/installation/kubeadm_settings.pm (hunk 1 above)
Comment (kravciak) [reply tests/installation/kubeadm_settings.pm]: Fixed

File: tests/installation/kubeadm_settings.pm (hunk 2 above)
Comment (kravciak) [reply tests/installation/kubeadm_settings.pm]: I prefer to always set the same ntp server so we can check if config was applied after installation

**Pull request merged**